src/main/java/seedu/modtracker/Module.java

```diff
@@ -73,4 +73,11 @@ public boolean equals(Object obj) {
         return moduleCode.equals(m.moduleCode);
     }
 
+    public boolean doesExpectedWorkLoadExist() {
+        return (expected != -1);
+    }
+
+    public boolean doesActualTimeExist(int weekNumber) {
+        return (actualTime[weekNumber] != -1);
+    }
```

Comment:
Should we change -1 into a variable called INVALID_WORKLOAD or INVALID_WEEK too?

Reply:
Sounds like a good idea!